build.sc

```diff
@@ -227,6 +227,8 @@ object cosim extends Module {
 
   class emulator(xLen: String) extends Module {
 
+    val ncores: Int = Runtime.getRuntime.availableProcessors()
```

Comment:
In case of large size cores machine, limit it to 8.

Reply:
Set it to 8 causes `%Error: /nix/store/n46a7nkil7clgvmljxw2cp7hwj40jczh-verilator-5.006/share/verilator/include/verilated.cpp:2499: VerilatedContext has 2 threads but model 'VTestBench' (instantiated as 'TOP') was Verilated with --threads 8.` in CI.
How about set it to 2?

Reply:
MAX(n, 8) should be fine.

Reply:
limit it to 8

Reply:
Now it's Min(nCores,8)